packages/checkout/sdk/src/smartCheckout/cancel/cancel.ts

```diff
@@ -54,16 +59,38 @@ export const cancel = async (
     );
   }
 
-  const result = await signFulfillmentTransactions(provider, [unsignedCancelOrderTransaction]);
-  if (result.type === SignTransactionStatusType.FAILED) {
+  if (overrides.waitFulfillmentSettlements) {
+    const result = await signFulfillmentTransactions(provider, [unsignedCancelOrderTransaction]);
```

Comment:
Sorry I should have caught it in the prev PR -- why aren't we passing a `waitSettlements` flag to `signFulfillmentTransactions` and let `signFulfillmentTransactions` deal with the logic so you don't have to duplicating the `signFulfillmentTransactions`'s logic on like 79?

Reply:
Yep good suggestion, I was wondering if we should be sending back the transactions on success as well.. I'll refactor to include that.